fonts/COPYING

```diff
@@ -0,0 +1,18 @@
+Copyright:
```

Comment:
There's no better solution than packaging the fonts directly in crnsimul?

Reply:
Not atm

Reply:
I will fix it when I find a better solution for a default font

Reply:
Does [this](https://stackoverflow.com/questions/10542832/how-to-use-fontconfig-to-get-font-list-c-c) look like a solution?

Reply:
This could be a solution, but it only works for Linux